### PR TITLE
aleph zero examples

### PR DESCRIPTION
**Pull request by antonefimow**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,14 +10,18 @@
   "dependencies": {
     "@bufbuild/protobuf": "^1.8.0",
     "@connectrpc/connect": "^1.4.0",
-    "@connectrpc/connect-web": "^1.4.0"
+    "@connectrpc/connect-web": "^1.4.0",
+    "@polkadot/api": "^11.0.3",
```
[the remark this answers is not in the document]
Comment (antonefimow) [reply package.json]: sure

File: src/examples/example-aleph-zero-linked.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
+import crypto from "crypto";
+import RampClient, {AlephZeroSignature, CosmosSignature, Signature} from "../";
+import {
+    GetAccountInfoRequest,
+    Protocol,
+    SetBankAccountRequest,
+    WhitelistAddressRequest
+} from "../gen/ramp/v1/public_pb";
+import * as process from "node:process";
```
Comment (MartinBechtle) [inline src/examples/example-aleph-zero-linked.ts:9]: unused import?

File: src/examples/example-aleph-zero-linked.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+import {ApiPromise, WsProvider} from '@polkadot/api';
+import {Keyring} from '@polkadot/keyring';
+import {blake2AsHex, blake2AsU8a, cryptoWaitReady} from '@polkadot/util-crypto';
+import * as secp256k1 from "secp256k1";
```
Comment (MartinBechtle) [inline src/examples/example-aleph-zero-linked.ts:14]: I see many of these are unused too

File: src/examples/example-aleph-zero-unlinked.ts [hunk 4]
```diff
@@ -0,0 +1,48 @@
+
```
Comment (MartinBechtle) [inline src/examples/example-aleph-zero-unlinked.ts:1]: Can you add instructions on top here and in the other file like with the Terra example? And also the exact npx command to run

**Review by MartinBechtle — DISMISSED** [review]
I ran `npm install` and `npx tsx src/examples/example-aleph-zero-linked.ts` and it worked perfectly. I'll approve pending the minor changes to imports and comments, no need to wait for re-approval.

**Pull request merged**